plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraClientConfig.java

```diff
@@ -55,6 +53,12 @@
 })
 public class CassandraClientConfig
 {
+    public enum CassandraAuthenticationType
+    {
+        NONE,
+        PASSWORD;
```

Comment:
nit: should we add  ` /**/;` int the end of enum declaration as the pseudo enum option to minimize diff in the future? i.e.
```
NONE,
PASSWORD,
/**/;
```

Reply:
`;` after `/**/` is redundant. IntelliJ will report it as unnecessary semicolon.